src/queries/BroomCat.js

```diff
@@ -5,7 +5,22 @@
 	let mangaOption = create("option",false,"Manga",select);
 	animeOption.value = "ANIME";
 	mangaOption.value = "MANGA";
-	createCheckbox(miscOptions,"restrictToList");
+	let sortSelect = create("select","#sortSelect",false,miscOptions);
+	let popOption = create("option",false,"Popularity",sortSelect);
+	let idOption = create("option",false,"Date Added",sortSelect);
+	popOption.value = "POPULARITY_DESC";
+	idOption.value = "ID_DESC";
+	let listRestrict = createCheckbox(miscOptions,"restrictToList");
+	listRestrict.onchange = function(){
```

Comment:
Why disable sorting here?

Reply:
There's two different queries and I was only focused on adding sort for the full media query. (They use different sort enums `MediaSort` vs `MediaListSort`)

I can look into adding (more) sort options for both in either a merged select that toggles available options or two different selects based on the state of the checkbox.

Reply:
Ah, forgot that those are different sort enums.